data/json/furniture_and_terrain/terrain-floors-outdoors.json

```diff
@@ -366,6 +366,25 @@
       "items": [ { "item": "splinter", "count": [ 2, 8 ] }, { "item": "nail", "charges": [ 5, 10 ] } ]
     }
   },
+  {
+    "type": "terrain",
+    "id": "t_floor_wooden_gangway",
+    "name": "wooden floor gangway",
+    "description": "Wooden gangway bridging a short gap.  Sturdy enough to act as a floor, but not sufficient to support additional flooring.",
+    "symbol": ".",
+    "color": "brown",
+    "looks_like": "t_floor",
+    "move_cost": 2,
+    "flags": [ "TRANSPARENT", "FLAMMABLE_HARD", "FLAT", "ROAD" ],
+    "bash": {
+      "sound": "SMASH!",
+      "ter_set": "t_null",
+      "str_min": 50,
+      "str_max": 400,
+      "str_min_supported": 100,
+      "items": [ { "item": "splinter", "count": [ 2, 8 ] }, { "item": "nail", "charges": [ 5, 10 ] } ]
```

Comment:
Could probably give more plank/splinters, seems excessive that bashing anihilate 14 planks completly to the points were only 8 splinter can be recovered.

Reply:
I agree with that logic. I've increased the resultant products somewhat arbitrarily. Since it was a direct copy of t_floor_noroof, I updated that bash result as well.